components/clp-package-utils/clp_package_utils/controller.py

```
@@ -1206,8 +1232,8 @@ def start(self) -> None:
             should_compose_project_be_running=False, project_name=self._project_name
         )
 
-        deployment_type = self._clp_config.get_deployment_type()
-        logger.info(f"Starting CLP using Docker Compose ({deployment_type} deployment)...")
+        orchestration_type = self._clp_config.compression_scheduler.type
+        logger.info(f"Starting CLP using Docker Compose ({orchestration_type} orchestration)...")
```

Ruff (0.15.18)

[warning] Logging statement uses f-string

(G004)